### DISCUSSION TITLE
issue? => add a discussions tab?

### DISCUSSION THREAD

**Discussion by ldijkman** [antoniandre/splitpanes#199] [General]
issue => add a discussions tab?

impressed what i see here
wish i had seen it earlier

treid a program that uses this i think 
codepen clone with ace editor

and played with 

https://plnkr.co/edit/Digj5Fw7D6hxaTU4

![2023-09-08-201046_1920x1080_scrot](https://github.com/prevue-land/ace-editor-codepen-clone/assets/45427770/918d7def-6529-4090-8fef-bef82659970c)

---

![2023-09-08-202152_1920x1080_scrot](https://github.com/prevue-land/ace-editor-codepen-clone/assets/45427770/b3780140-fe08-4a4c-a821-8e25449ad846)

**Comment by antoniandre** [comment]
Thanks for your feedback!
